lpc55_sign/src/lib.rs

```diff
@@ -195,4 +198,7 @@ pub enum Error {
 
     #[error("fmt error: {0}")]
     FmtError(#[from] std::fmt::Error),
+
+    #[error(transparent)]
+    TryFromSliceError(#[from] std::array::TryFromSliceError),
```

Comment:
Vibes: I don't really like making this an automatically-constructed error type, and prefer `try_from().unwrap()` even though it's wordier.

We only ever do `try_from()` using constant indices, so a failure always indicates a programming error and no meaningful error handling can occur.

Reply:
I think I've developed a slight allergy to `unwrap`; I basically think every occurrence is a bug waiting to bite me at run-time. But of course you're quite right in this case; I just wish there were a better idiom. Reverted in 57dc178.